### PR TITLE
define write and read-only databases

### PR DESCRIPTION
**Pull request by alfredodeza**
Mostly playbook changes and fixes to be able to deploy both the app and the databases.

### CODE REVIEW

File: deploy/playbooks/roles/common/tasks/postgresql.yml [hunk 1]
```diff
@@ -5,52 +5,24 @@
...
 
+- set_fact:
+    postgresql_app_user_password: "{{ generated_app_user_password.stdout }}"
+
 - name: Make {{ app_name }} user
   postgresql_user:
     name: "{{ app_name }}"
-    password: "{{ db_password.stdout }}"
-    role_attr_flags: SUPERUSER
+    password: "{{ generated_app_user_password.stdout }}"
```
Comment (andrewschoen) [inline deploy/playbooks/roles/common/tasks/postgresql.yml]: You should use ``postgresql_app_user_password`` here.

File: deploy/playbooks/roles/common/templates/prod_db.py.j2 [hunk 2]
```diff
@@ -1,7 +1,15 @@
 # {{ ansible_managed }}
 
-sqlalchemy = {
-    'url': 'postgresql+psycopg2://{{ app_name }}:{{ db_password.stdout }}@127.0.0.1/{{ app_name }}',
+sqlalchemy_w = {
+    'url': 'postgresql+psycopg2://{{ app_name }}:{{ postgresql_app_user_password }}@{{ master_ip }}/{{ app_name }}',
```
Comment (andrewschoen) [inline deploy/playbooks/roles/common/templates/prod_db.py.j2]: I thought we wanted to use hostnames here instead of IP addresses? IIRC, that was so we could swap these servers by making ``/etc/hosts`` changes instead of redeploying the app.

Comment (alfredodeza) [inline deploy/playbooks/roles/common/templates/prod_db.py.j2]: true. I decided to leave that to a separate PR so that I could have something that already works

File: deploy/playbooks/roles/common/tasks/postgresql.yml (hunk 1 above)
Comment (alfredodeza) [inline deploy/playbooks/roles/common/tasks/postgresql.yml]: I'll fix. Good catch

File: deploy/playbooks/roles/common/templates/prod_db.py.j2 (hunk 2 above)
Comment (andrewschoen) [inline deploy/playbooks/roles/common/templates/prod_db.py.j2]: Ok, that sounds reasonable.

**Comment by andrewschoen** [comment]
It looks like the tests need updated to support the new database setup.

https://jenkins.ceph.com/job/shaman-pull-requests/20/console
